src/mod/docs/index.md

```diff
@@ -75,21 +79,20 @@ Moderation commands are available through Cliptok and Dyno. Some commands are re
 
 Events are logged in the seven logging channels as well as the Discord Audit Log.
 
-### #cliptok-user-logs
-
+::: details #cliptok-user-logs {#cliptok-user-logs}
```

Comment:
This is already an advanced page for mod reference, I don't see any reason to hide details behind collapsible boxes. You also do this for the logs but not for commands, which is strange.